docs/build/build-smart-contracts.md

```diff
@@ -198,11 +199,19 @@ Try deploying a smart contract to Moonbeam by following their [documentation][mo
 
 #### Astar
 
-[Astar Network](https://astar.network/) supports the building of dApps with EVM and WASM smart contracts and offers developers true interoperability. True interoperability with cross-consensus messaging [XCM](https://wiki.polkadot.network/docs/learn-xcm) and cross-virtual machine [XVM](https://github.com/AstarNetwork/). We are made by developers and for developers. Astar’s unique Build2Earn model empowers developers to get paid through a dApp staking mechanism for the code they write and dApps they build.
+[Astar Network](https://astar.network/) supports the building of dApps with EVM and WASM smart
```

Comment:
I think unformatted (not pretty) code still sneaks in to the repo from time to time when changes are made directly via the GitHub UI.

Reply:
Got it! Thanks!